### ISSUE TITLE
Initial VM evolution

### ISSUE THREAD

**Issue by rmcewan** [nlpie/nlp-adapt#2]
Keep existing VM architecture/approach and add the following components:

- [x] install ctakes 
- [x] install metamap - 2016 release; requires plugin to have it export UIMA formatted data
~~install clamp - SVP will contact Hua for permission (still waiting on communications)~~

**Comment by rfinz** [comment]
Partially complete, partially blocked

**Comment by rfinz** [comment]
Metamap requires additional heap allocation. I'm adding it to JAVA_TOOL_OPTIONS in the environment configuration script. The UIMA wrapper seems to be working.

**Comment by rfinz** [comment]
Ack, Metamap with the UIMA Wrapper still experiences an OutOfMemoryError on file 554. That result seems consistent.

```
Can't load log handler "java.util.logging.FileHandler"
java.nio.file.AccessDeniedException: uima.log.lck
java.nio.file.AccessDeniedException: uima.log.lck
	at sun.nio.fs.UnixException.translateToIOException(UnixException.java:84)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:102)
	at sun.nio.fs.UnixException.rethrowAsIOException(UnixException.java:107)
	at sun.nio.fs.UnixFileSystemProvider.newFileChannel(UnixFileSystemProvider.java:177)
	at java.nio.channels.FileChannel.open(FileChannel.java:287)
	at java.nio.channels.FileChannel.open(FileChannel.java:335)
	at java.util.logging.FileHandler.openFiles(FileHandler.java:459)
	at java.util.logging.FileHandler.<init>(FileHandler.java:263)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at java.util.logging.LogManager$5.run(LogManager.java:966)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.util.logging.LogManager.loadLoggerHandlers(LogManager.java:958)
	at java.util.logging.LogManager.initializeGlobalHandlers(LogManager.java:1578)
	at java.util.logging.LogManager.access$1500(LogManager.java:145)
	at java.util.logging.LogManager$RootLogger.accessCheckedHandlers(LogManager.java:1667)
	at java.util.logging.Logger.getHandlers(Logger.java:1777)
	at java.util.logging.Logger.log(Logger.java:735)
	at java.util.logging.Logger.doLog(Logger.java:765)
	at java.util.logging.Logger.logp(Logger.java:931)
	at org.apache.uima.util.impl.JSR47Logger_impl.logrb(JSR47Logger_impl.java:365)
	at org.apache.uima.collection.impl.cpm.engine.CPMThreadGroup.uncaughtException(CPMThreadGroup.java:77)
	at java.lang.Thread.dispatchUncaughtException(Thread.java:1959)
java.lang.OutOfMemoryError: Java heap space
	at java.lang.StringCoding$StringEncoder.encode(StringCoding.java:300)
	at java.lang.StringCoding.encode(StringCoding.java:344)
	at java.lang.String.getBytes(String.java:918)
	at org.apache.uima.tools.components.InlineXmlCasConsumer.processCas(InlineXmlCasConsumer.java:158)
	at org.apache.uima.analysis_engine.impl.compatibility.CasConsumerAdapter.process(CasConsumerAdapter.java:99)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.callAnalysisComponentProcess(PrimitiveAnalysisEngine_impl.java:401)
	at org.apache.uima.analysis_engine.impl.PrimitiveAnalysisEngine_impl.processAndOutputNewCASes(PrimitiveAnalysisEngine_impl.java:318)
	at org.apache.uima.analysis_engine.impl.AnalysisEngineImplBase.process(AnalysisEngineImplBase.java:269)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.processNext(ProcessingUnit.java:895)
	at org.apache.uima.collection.impl.cpm.engine.ProcessingUnit.run(ProcessingUnit.java:575)
```
**UPDATE** Despite the cryptic nature of this error, it seems like it's just metamap choking on particular files. I'll update the mimic data issue with examples.

**Comment by rfinz** [comment]
Emailed Serguei about CLAMP - going to consider this closed and CLAMP as a stretch goal unless notified otherwise
